src/components/misc/StaticMap.jsx

```diff
@@ -17,39 +19,66 @@ export default class StaticMap extends React.Component {
     };
 
     render() {
-        let loc = this.props.location;
-        let lat = loc.lat;
-        let lng = loc.lng;
-
-        const imgSrc = url.format({
-            protocol: 'https',
-            hostname: 'maps.googleapis.com',
-            pathname: '/maps/api/staticmap',
-            query: {
-                center: (lat + 0.0002) + ',' + lng,
-                markers: lat + ',' + lng,
-                zoom: 17,
-                size: this.props.width + 'x' + this.props.height,
-                key: 'AIzaSyAHVagqI3RTd0psf57oA6gzKqVyjp8FS8w',
-            }
-        });
-
-        // Convert lat/lng to degrees, minutes and seconds, which is more
-        // readable than the decimal value with tons of decimals.
-        let latDMS = convertLatToDMS(lat), lngDMS = convertLngToDMS(lng);
-        let latStr = latDMS.deg.toString()
-            .concat('° ', latDMS.min, '\' ', latDMS.sec, '" ', latDMS.dir);
-        let lngStr = lngDMS.deg.toString()
-            .concat('° ', lngDMS.min, '\' ', lngDMS.sec, '" ', lngDMS.dir);
+        let placeholder = this.props.placeholder;
+        let map = null;
+        let coordinates = null;
 
-        return (
-            <div className="StaticMap"
-                onClick={ this.onClick.bind(this) }>
+        if (placeholder) {
```

Comment:
This should not be based on some special boolean value, but on whether or not there is a location (i.e. whether `this.props.location` is null).